lib/ansible/modules/cloud/amazon/ec2_vpc.py

```diff
@@ -730,6 +756,9 @@ def main():
     else:
         module.fail_json(msg="region must be specified")
 
+    if not module.params.get('vpc_id') and not module.params.get('resource_tags'):
```

Comment:
Because it's not at all documented and I'd forgotten about its existence? (even the ec2 helper functions are better documented than `required_one_of`). I've updated the PR now to use it.